openqabot/types/incidents.py

```diff
@@ -54,7 +54,7 @@ def _is_scheduled_job(
                 f"{QEM_DASHBOARD}api/incident_settings/{inc.id}",
                 headers=token,
             ).json()
-        except Exception as e:  # pylint: disable=broad-except
+        except Exception as e:
```

Comment:
this seems to have been lost in rebasing

Reply:
Linter doesn't complain though